images/compose/compose.folders.yml

```diff
@@ -1,30 +1,46 @@
-version: '3'
+# This file is intended as a starting point. Before using it, make a copy and adjust at least the passwords.
+---
+version: '3.7'
```

Comment:
Is there any reason why the version is now 3.7?

Reply:
Not part of this pr anymore...
Also not a particular reason. Just from examples.